### PR TITLE
Added Effect: "Repossession"

### PR DESCRIPTION
**Pull request by ubscal**
Requested from https://github.com/gta-chaos-mod/ChaosModV/issues/1217#issue-870530370

### CODE REVIEW

**Comment by Last0xygen** [comment]
This doesn't do anything, if the player is not in any vehicle though. :/
Maybe spawn Jizzles car if not in any vehicle?

**Comment by ubscal** [comment]
> This doesn't do anything, if the player is not in any vehicle though. :/
> Maybe spawn Jizzles car if not in any vehicle?

Yea that seems like a good idea, I'll add that in a couple hours, kinda busy right now.

**Comment by ubscal** [comment]
> This doesn't do anything, if the player is not in any vehicle though. :/
> Maybe spawn Jizzles car if not in any vehicle?

Now spawns Jizzle's car if the player is not in a vehicle

File: ConfigApp/Effects.cs [hunk 1]
```diff
@@ -588,6 +589,7 @@ public enum EffectType
             {EffectType.EFFECT_PLAYER_AIMBOT, new EffectInfo("Aimbot", EffectCategory.PLAYER, "player_aimbot", true)},
             {EffectType.EFFECT_PEDS_SPAWN_BIKER, new EffectInfo("Spawn Biker", EffectCategory.PEDS, "peds_spawn_biker")},
             {EffectType.EFFECT_PEDS_SPAWN_SPACE_RANGER, new EffectInfo("Spawn Space Ranger", EffectCategory.PEDS, "peds_spawn_space_ranger")},
+            {EffectType.EFFECT_REPOSSESSION, new EffectInfo("Repossession", EffectCategory.VEHICLE, "veh_repossession", true, true) },
```
Comment (pongo1231) [inline ConfigApp/Effects.cs:592]: Any reason for the `IsTimed` and `IsShortTimed` flags here?

Comment (ubscal) [reply ConfigApp/Effects.cs:592]: Yea, I was testing another effect I made locally and I guess I just changed the effect name to "EFFECT_REPOSSESSION" when I saw the request for this. I only replaced the effect name and I guess my dumbo brain forgot to change the flags. I'll fix that now.

**Pull request merged**